// try catch

let num1 = 5;
let num2 = 10;

console.log(`num1 = ${num1}, num2 = ${num2}`);
try {
  console.log(`num1 = ${num1}, num2 = ${num2}, num3 = ${num3}`);
} catch (err) {
  console.log(err);
}
